pipelines/jobs/configurations/jdk21u_evaluation.groovy

```diff
@@ -1,7 +1,7 @@
 targetConfigurations = [
-        'aarch64Windows' : [
-                'temurin'
-        ]
+        // 'aarch64Windows' : [
```

Comment:
Not just remove this?

Reply:
I figured leaving the block was useful as an example for people who want to add new platforms otherwise it's just empty but I'm also happy to remove it. Note that the comments are quite common in other files. E.g https://github.com/adoptium/ci-jenkins-pipelines/blob/master/pipelines/jobs/configurations/jdk11u_evaluation.groovy#L9

Reply:
Would probably prefer a permanent comment block at the top explaining how the section is used in that case, looks deliberate otherwise.